custom_components/tuya_local/devices/pir_sensor_alarm.yaml

```diff
@@ -0,0 +1,69 @@
+name: PIR sensor alarm
+products:
+  - id: y5tdmwmb2ieknmkb
+entities:
+  - entity: binary_sensor
+    translation_key: motion
+    class: motion
+    dps:
+      - id: 1
+        type: string
+        name: sensor
+        mapping:
+          - dps_val: pir
+            value: true
+          - dps_val: "none"
```

Comment:
It is probably more failsafe to just make false the default (remove the dps_val filter from this list item)